Readme.md

```diff
@@ -1,206 +1,210 @@
 
-An `accepted` sign-off means this bug is fixed and will be in the bug fix list
-of next release. The developer helped verify the bug should follow the steps:
+1. collect analytic data;
+2. interview users;
 
-1. Squash all the commits from PR page into 1 commit with a graceful message.
-2. In the commit message, we should have the following information to be set:
-  1. `Fixes #<PR-NUM>` in this way, we are able to quickly find the related PR from commit and find the related commits from PR as well.
-  2. `CARD: <URL>` reference to the card url in bugs board
-  3. `Reviewed-By: @<github-username>` this flags who reviewed this PR
-3. Submit a comment on the related card with the hash string of hotfix commit and related PR.
+to back up his (or someone else's) request of this enhancement.
 
-Therefore, the entire example of a squashed commit looks like:
+He needs to reason why that
 
-```
-Don't send an email to operator
-[other messages...]
+1. current design is not the best;
+2. it worth time and money to do this improvement;
 
-Fixes #12
-CARD: https://trello.com/xxxxxx
-Reviewed-By: @yorkie
-```
+and compile them into a `user story` to help his co-workers to understand what
+exact problem this enhancement is to solve.
 
-The example card comment:
 
-```
-This bug has been fixed via https://github.com/weflex/product-development-guide/commit/cf96be684c8c5c79618260d10418795c0736c69b.
+#### Difference between enhancements and technical issues
 
-Related PR: https://github.com/weflex/product-development-guide/pull/1
-```
+The major difference between enhancements and technical issues is that
+enhancements emphysis on _not properly designed_ while the latter one emphsis on
+_not fully designed_.
 
-(Note: 1st line of message should be less than 50 characters)
 
-If `rejected`, card will be push back to `InProgress` again and assignee has to
-re-fix it.
+### New Features
 
-### Enhancements
+> Ah! A feature!
+
+New features are usually new business problems. If a feature or function is
+designed and implemented at all, then it probably be a new feature (to work on).
 
-![enhancements-board](images/enhancements-board.png)
+A feature requires a user story, similar to the
+[enhanment one](#user-stories-for-enhancements).
 
-Similar to the `Bugs`, the `Enhancements` board also come with four lists,
-`NextUp`, `InProgress`, `Staging` and `Live`. Each card in `Enhancements`
-travels from the left-most `NextUp` waitlist and ends up in going `Live`.
 
-A card moves into `Staging` list means it coding (if needed) is done and now
-it's time to verify its functions. This step would usually take a couple of
-days to a week.
 
-If the feature is correct (function-wise) and stable, we will move the card to
-`Live` and put it into production.
+Review Days
+----
 
-`Work In Progress` items in `Enhancement` workflow may be in these types:
+Review day is a meeting, hold at end of each cycle, to report and demo what
```

Comment:
Actually could we create a `meetings` directory under `git@github.com:weflex/dev` to store the meeting records, the product manage should be responsible to handle :)

Basically it likes https://github.com/nodejs/node/tree/4b1c27f94edcbdb8a3b2d8f3532eb452f5a319eb/doc/tsc-meetings